packages/php/email-editor/src/Integrations/Core/Renderer/Blocks/class-audio.php

```
@@ -0,0 +1,166 @@
+			$parsed_block['email_attrs'] ?? array()
+		);
+	}
+	/**
+	 * Renders the audio block content as an audio player for email.
+	 *
+	 * @param string            $block_content Block content.
+	 * @param array             $parsed_block Parsed block.
+	 * @param Rendering_Context $rendering_context Rendering context.
+	 * @return string
+	 */
+	protected function render_content( string $block_content, array $parsed_block, Rendering_Context $rendering_context ): string { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

PHPMD (2.15.0)

Avoid unused parameters such as '$rendering_context'. (undefined)

(UnusedFormalParameter)